public/app/features/templating/templateSrv.js

```diff
@@ -107,10 +111,15 @@ function (angular, _) {
       });
     };
 
-    this.getAllValue = function(variable) {
+    this.getAllValue = function(variable, format) {
       if (variable.allValue) {
         return variable.allValue;
       }
+
```

Comment:
@torkelo (val1|val2|val3) doesn't work with 1000 servers, you will hit 414 Request-URI Too Large error pretty fast.

Custom allValue doesn't work at all.

Also, isn't the point of All value to match ALL values? Why should you send all values as a gigantic regex?